src/libs/API.js

```diff
@@ -29,6 +30,10 @@ function write(command, apiCommandParameters = {}, onyxData = {}) {
         ...apiCommandParameters,
         appversion: pkg.version,
         apiRequestType: CONST.API_REQUEST_TYPE.WRITE,
+
+        // We send the pusherSocketID with all write requests so that the api can include it in push events to prevent Pusher from sending the events to that client. The push event is sent
```

Comment:
instead of `that client` I think it'd be better if you specified that it was the caller

Reply:
Updated